resources/charts/bitcoincore/charts/lnd/templates/pod.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "lnd.fullname" . }}
+  labels:
+    {{- include "lnd.labels" . | nindent 4 }}
+    {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 4 }}
+    {{- end }}
+    app: {{ include "lnd.fullname" . }}
+    {{- if .Values.collectLogs }}
+    collect_logs: "true"
+    {{- end }}
+    chain: {{ .Values.global.chain }}
+spec:
+  {{- with .Values.imagePullSecrets }}
+  imagePullSecrets:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  securityContext:
+    {{- toYaml .Values.podSecurityContext | nindent 4 }}
+  containers:
+    - name: {{ .Chart.Name }}
+      securityContext:
+        {{- toYaml .Values.securityContext | nindent 8 }}
+      image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+      imagePullPolicy: {{ .Values.image.pullPolicy }}
+      ports:
+        - name: rpc
+          containerPort: {{ .Values.RPCPort }}
+          protocol: TCP
+        - name: p2p
+          containerPort: {{ .Values.P2PPort }}
+          protocol: TCP
+        - name: rest
+          containerPort: {{ .Values.RestPort }}
+          protocol: TCP
+      livenessProbe:
+        {{- toYaml .Values.livenessProbe | nindent 8 }}
+      readinessProbe:
+        {{- toYaml .Values.readinessProbe | nindent 8 }}
+      startupProbe:
+        {{- toYaml .Values.startupProbe | nindent 8 }}
+      resources:
+        {{- toYaml .Values.resources | nindent 8 }}
+      volumeMounts:
+        {{- with .Values.volumeMounts }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        - mountPath: /root/.lnd/lnd.conf
+          name: config
+          subPath: lnd.conf
+        - mountPath: /root/.lnd/tls.key
+          name: tlskey
+          subPath: tls.key
+        - mountPath: /root/.lnd/tls.cert
+          name: tlscert
+          subPath: tls.cert
+    {{- if .Values.circuitBreaker }}
```

Comment:
Is the plan to have this removed before this PR is merged? Or shortly after? Circuit breaker feels like a ~service~ plugin.

Reply:
good catch, this is leftover from Italy. ill need circuit breaker together with simln eventually either in this PR or a follow up, so this'll get cleaned up before merge